.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea
+vendor
+.terraform
+.vscode
+*.tfstate
+*.tfstate.backup
```

Comment:
Thanks for adding a gitignore! Could you please remove non-Terraform files from it? You can ignore these globally if you prefer with:

```
echo ".idea" >> ~/.gitignore
echo ".vscode" >> ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```